frontend/src/metabase/visualizations/visualizations/PivotTable.jsx

```diff
@@ -393,6 +427,35 @@ export default class PivotTable extends Component {
   }
 }
 
+function RowToggleIcon({
```

Comment:
@mazameli if you want to do a styling pass, this component is the +/- icons.

Reply:
Perfect, thanks.

Reply:
Added some styling to it. 👌🏼